### ISSUE TITLE
Mid-level markdown editor

### ISSUE THREAD

**Issue by justinkelly** [standardnotes/web#100]
Another feature proposal

I would be great it the there was a mid-level markdown editor/extension (more than simple - less than advanced) that had the features of

* Colourized markdown editor (like the code editor set to markdown)
* No render preview (like the code editor)
* No language option - just permanently set to Markdown (like Simple Markdown editor)
* No line numbers (like Simple Markdown editor)
* That wraps long lines (like simple and advanced editor)
* That doesn't add text styling (larger letters for headings, etc. )

basically codemirror set to markdown

Thanks again - keep up the amazing work on StandardNotes - I really appreciate this amazing project!!!

**Comment by justinkelly** [comment]
The below gist is a fork of https://github.com/standardnotes/extensions-server/blob/master/app/views/editors/code.html.erb but with the changes listed above

https://gist.github.com/justinkelly/cdf57ee55799fb092364543d11f82a64

**Comment by mobitar** [comment]
I like this idea. There's a few urgent items ahead of this on my list, but I'll try to release something official soon.

**Comment by justinkelly** [comment]
thanks mo - no rush - thanks again for this great project!!

**Comment by mobitar** [comment]
See minimal markdown editor.

**Comment by justinkelly** [comment]
thanks !! am now using minimal as my default editor!!
